src/common/Finisher.cc

```diff
@@ -61,23 +58,27 @@ void *Finisher::finisher_thread_entry()
       ldout(cct, 10) << "finisher_thread doing " << ls << dendl;
 
       // Now actually process the contexts.
+      utime_t start;
```

Comment:
if set start here, it would not be updated in each "for" iteration. Is it expected?